packages/server/src/index.ts

```diff
@@ -1 +1,35 @@
-export default {};
+import {KeyType, Property, Signature} from "../../message/src/index";
+
+type SecretKey = {
+  type: KeyType;
+  key: string;
+}
+
+type PublicKey = {
+  type: KeyType;
+  key: string;
+}
+
+// TODO: @ts-ignore should be removed, when below function is implemented.
+// @ts-ignore
+export function auth_privacy(properties: Property[], secret: SecretKey): Signature {
```

Comment:
I think I told you to let us use `camelCase` notation as our naming convention.
How do you think about it?

Reply:
Fixed it.